src/frontend/base/default/ebanx/js/instalments.js

```diff
@@ -0,0 +1,27 @@
+/* global ebanxUpdateFireInterest */
+
+const initInstalments = (code, lowerCountry) => { // eslint-disable-line no-unused-vars
+  if (typeof ebanxUpdateFireInterest !== 'undefined') {
+    document.querySelector(`#${code}_instalments`)
```

Comment:
What if there's no element with the id `${code}_instalments`?

Reply:
I checked in the end of file "if (selectInstalment) {"